Documentation/network/servicemesh/gateway-api/mistakes-warning.rst

```diff
@@ -0,0 +1,4 @@
+.. warning::
+
+   Gateway API is a recent addition to Kubernetes. As it is not yet widely used we have not yet had a chances to get lots of user feedback.
+   If you ran into an issue which was a mistake and not yet listed here we highly appreciate a PR to add yours to the list!
```

Comment:
"We" is unclear, specify as "the Cilium project"; "mistake" is unclear; use present tense; use clear, plain language
```suggestion
   Gateway API is a recent addition to Kubernetes, and the Cilium project has not yet received much user feedback.
   If you encounter an issue that is not yet listed in this section, consider opening a PR to add yours to the list.
```